triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceConfig.scala

```diff
@@ -134,6 +138,22 @@ private[trigger] object ServiceConfig {
       .text(
         s"Maximum time interval between restarting a failed trigger. Defaults to ${DefaultMaxRestartInterval.toSeconds} seconds.")
 
+    opt[Int]("max-http-entity-upload-size")
+      .action((x, c) => c.copy(maxHttpEntityUploadSize = x))
+      .optional()
+      .text(
+        s"Optional max HTTP entity upload size. Defaults to ${DefaultMaxHttpEntityUploadSize}. HTTP uploads cannot be streamed but must be fully uploaded when a request is deferred until after a login flow with the auth middleware completed.")
+      // TODO[AH] Expose once the auth feature is fully implemented.
+      .hidden()
```

Comment:
The auth feature is not yet exposed (`.hidden`) so it doesn't make sense to expose these flags independently.